README.md

```diff
@@ -1,16 +1,19 @@
 Developer guide to Tessel 2 Docs Gitbook
 ============
 
-# Tools
-Install the following command line tools using `npm i {{toolname}} -g`
+# Requirements
+The docs are build using a command line tool called [GitBook](https://www.npmjs.com/package/gitbook), which requires:
 
-* [gitbook](https://www.npmjs.com/package/gitbook)
```

Comment:
I addressed this [here](https://github.com/tessel/t2-docs/pull/77#discussion_r68756925)